### ISSUE TITLE
Cannot install RiderLink plugin in Unreal 4.26

### ISSUE THREAD

**Issue by MoRunChang2015** [JetBrains/UnrealLink#71]
Hi，

When I try to install RiderLink plugin,  I got some errors like this:

> 17:21	Failed to build RiderLink plugin: Failed to build plugin for G:\GrandStudyAuto\UE4_Porject\QuickStart\QuickStart.uproject (show balloon)

> 17:21	Failed to build RiderLink plugin
			Check build logs for more info
			Help > Diagnostic Tools > Show Log in Explorer
			And contact dev team for help at GitHub Issues page

And then,  I try this one ：

try this one:

1. Find RiderLink.zip in Rider for Unreal Engine installation folder;
example: if Rider is installed in D:\JetBrains\Rider for Unreal Engine 2020.2, RiderLink.zip will be at D:\JetBrains\Rider for Unreal Engine 2020.2\plugins\UnrealLink\EditorPlugin\RiderLink.zip. To find Rider installation folder, please use the menuHelp | Diagnostic Tools | Special Files and Folders or Help | Diagnostic Tools | Browse special paths....
2. Unzip RiderLink.zip to any place outside of Unreal Engine installation or Game project
example C:/tmp/RiderLink
3. Run from cmd:
{EngineRoot}/Engine/Build/BatchFiles/RunUAT.bat BuildPlugin -Plugin="C:/tmp/RiderLink/RiderLink.uplugin" -Package="C:/tmp/RiderLink_Package" -Rocket
4. Check if it builds or not.
4.1 If it doesn't build, send build logs here
4.2 If it builds fine, try to place plugin into {GameProject}/Plugins/Developer/RiderLink folder and try to run project

It still build failed. this is the log:

```
XGE.ExecuteTaskFile: Distributing 47 actions to XGE
<>c__DisplayClass24_0.<ExecuteTaskFileWithProgressMarkup>b__0: Maximum number of concurrent builds reached.
UnrealBuildTool.Main: CompilationResultException: Error: OtherCompilationError
UnrealBuildTool.Main:    at UnrealBuildTool.ActionGraph.ExecuteActions(BuildConfiguration BuildConfiguration, List 1 ActionsToExecute) at G:\GrandStudyAuto\UnrealEngine\Engine\Source\Programs\UnrealBuildTool\System\ActionGraph.cs:line 230
UnrealBuildTool.Main:    at UnrealBuildTool.BuildMode.Build(TargetMakefile[] Makefiles, List 1 TargetDescriptors, BuildConfiguration BuildConfiguration, ISourceFileWorkingSet WorkingSet, BuildOptions Options, FileReference WriteOutdatedActionsFile)  G:\GrandStudyAuto\UnrealEngine\Engine\Source\Programs\UnrealBuildTool\Modes\BuildMode.cs:line 483
UnrealBuildTool.Main:    at UnrealBuildTool.ExternalExecution.ExecuteHeaderToolIfNecessary(BuildConfiguration BuildConfiguration, FileReference ProjectFile, String TargetName, TargetType TargetType, Boolean bHasProjectScriptPlugin, List 1 UObjectModules, FileReference ModuleInfoFileName, Boolean bIsGatheringBuild, Boolean bIsAssemblingBuild, ISourceFileWorkingSet WorkingSet) G:\GrandStudyAuto\UnrealEngine\Engine\Source\Programs\UnrealBuildTool\System\ExternalExecution.cs:line 1160
UnrealBuildTool.Main:    at UnrealBuildTool.UEBuildTarget.Build(BuildConfiguration BuildConfiguration, ISourceFileWorkingSet WorkingSet, Boolean bIsAssemblingBuild, List1 SpecificFilesToCompile) G:\GrandStudyAuto\UnrealEngine\Engine\Source\Programs\UnrealBuildTool\Configuration\UEBuildTarget.cs:line 1729
UnrealBuildTool.Main:    at UnrealBuildTool.BuildMode.CreateMakefile(BuildConfiguration BuildConfiguration, TargetDescriptor TargetDescriptor, ISourceFileWorkingSet WorkingSet) G:\GrandStudyAuto\UnrealEngine\Engine\Source\Programs\UnrealBuildTool\Modes\BuildMode.cs:line 615
UnrealBuildTool.Main:    at UnrealBuildTool.BuildMode.Build(List1 TargetDescriptors, BuildConfiguration BuildConfiguration, ISourceFileWorkingSet WorkingSet, BuildOptions Options, FileReference WriteOutdatedActionsFile, Boolean bSkipPreBuildTargets)  G:\GrandStudyAuto\UnrealEngine\Engine\Source\Programs\UnrealBuildTool\Modes\BuildMode.cs:line 257
UnrealBuildTool.Main:    at UnrealBuildTool.BuildMode.Execute(CommandLineArguments Arguments) G:\GrandStudyAuto\UnrealEngine\Engine\Source\Programs\UnrealBuildTool\Modes\BuildMode.cs:line 226
UnrealBuildTool.Main:    at UnrealBuildTool.UnrealBuildTool.Main(String[] ArgumentsArray) G:\GrandStudyAuto\UnrealEngine\Engine\Source\Programs\UnrealBuildTool\UnrealBuildTool.cs:line 550
```

**Comment by daltonbr** [comment]
Having a similar issue.

**Comment by jhartikainen** [comment]
I ran into this problem as well, but in my case the problem was SmoothSync, which is a marketplace asset I had installed into the engine. I removed SmoothSync, and RiderLink installed just fine after that.

This seems like a potential issue in SmoothSync, so probably not something JetBrains needs to worry about - but figured I'd post this in case it helps someone.

In case it's useful, this was the error I saw that pointed me to this problem:

```D:/ohjelmat/UE_4.26/Engine/Plugins/Marketplace/SmoothSync/Source/SmoothSyncPlugin/Public/SmoothSync.h(473) : LogCompile: Error: An explicit Category specifier is required for any property exposed to the editor or Blueprints in an Engine module.```
